docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/artifactory/index.md

```diff
@@ -0,0 +1,142 @@
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
+## Configuring the Receiver
+
+An Artifactory webhook receiver must reference a Kubernetes `Secret` resource 
+with a `secret-token` key in its data map. This
+[shared secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by
+Artifactory to sign requests and by the receiver to verify those signatures.
+
+:::note
+The following commands are suggested for generating and base64-encoding a
+complex secret:
+
+```shell
+secret_token=$(openssl rand -base64 48 | tr -d '=+/' | head -c 32)
+echo "Secret token: $secret_token"
+echo "Encoded secret token: $(echo -n $secret_token | base64)"
+```
+:::
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Project
+metadata:
+  name: kargo-demo
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: artifactory-wh-secret
+  namespace: kargo-demo
+  labels:
+    kargo.akuity.io/cred-type: generic
+data:
+  secret-token: <base64-encoded secret token>
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: artifactory-wh-receiver
+      artifactory:
+        secretRef:
+          name: artifactory-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+Kargo will generate a hard-to-guess URL from the receiver's configuration. This
+URL can be obtained using a command such as the following:
+
+```shell
+kubectl get projectconfigs kargo-demo \
+  -n kargo-demo \
+  -o=jsonpath='{.status.webhookReceivers}'
+```
+
+## Registering with Artifactory
+
+1. Navigate to 
+   `https://<jfrog-instance>.jfrog.io/ui/admin/configuration/webhooks`, where
+   `<jfrog-instance>` has been replaced with na Artifactory instance for which 
+   you are an administrator.
+
+1. Click <Hlt>New Webhook</Hlt>.
+
+   ![Webhooks Dashboard](./img/webhooks.png "Webhooks Dashboard")
+
+1. Complete the <Hlt>Create new webhook</Hlt> form:
+
+   ![Add Webhook](./img/add-webhook.png "Add Webhook")
+
+1. Enter a descriptive name in the <Hlt>Name</Hlt> field.
+
+1. Complete the <Hlt>URL</Hlt> field using the URL
+   [for the webhook receiver](#retrieving-the-receivers-url).
+  
+1. Under <Hlt>Execution Results</Hlt> check <Hlt>Show status of successful 
+   executions in the Troubleshooting tab</Hlt>.
+
+   :::info
+   Although Artifactory supports sending test/dummy events to the URL,
+   only organically triggered events will show up in the troubleshooting tab.
+   Not test/dummy events. Even if they're successful.
+   :::
+
+1. Scroll down to <Hlt>Events</Hlt> and select <Hlt>Docker and OCI</Hlt> > 
+   <Hlt>Tag was pushed</Hlt>.
+
+   ![Select Trigger](./img/select-trigger.png "Select Trigger")
+
+1. Upon clicking out of the input menu, an <Hlt>Add Repositories</Hlt> modal 
+   will appear.
```

Comment:
We should try to keep steps imperative. Wording like "Complete the form in the modal dialog that appears:"

The next three steps can probably be indented to show they're part of this.